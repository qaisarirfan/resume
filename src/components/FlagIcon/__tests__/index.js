import React from "react"
import createShallow from "@material-ui/core/test-utils/createShallow"
import FlagIconCss from "../FlagIcon"
import renderer from "react-test-renderer";

const createProps = () => ({
  code: "en"
})

describe("<FlagIconCss />", () => {
  it("Does not explode", () => {
    const props = createProps()
    const component = createShallow()(<FlagIconCss {...props} />)
    expect(component).toMatchSnapshot()
  })

  it("Should render properly", () => {
    const props = createProps()
    const component = renderer.create(<FlagIconCss {...props} />)
    expect(component).toMatchSnapshot()
  })
})
