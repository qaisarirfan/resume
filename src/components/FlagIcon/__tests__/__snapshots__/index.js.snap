// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlagIconCss /> Does not explode 1`] = `
<FlagIcon
  classes={
    Object {
      "ad": "FlagIcon-ad-4",
      "adSquared": "FlagIcon-adSquared-5",
      "ae": "FlagIcon-ae-6",
      "aeSquared": "FlagIcon-aeSquared-7",
      "af": "FlagIcon-af-8",
      "afSquared": "FlagIcon-afSquared-9",
      "ag": "FlagIcon-ag-10",
      "agSquared": "FlagIcon-agSquared-11",
      "ai": "FlagIcon-ai-12",
      "aiSquared": "FlagIcon-aiSquared-13",
      "al": "FlagIcon-al-14",
      "alSquared": "FlagIcon-alSquared-15",
      "am": "FlagIcon-am-16",
      "amSquared": "FlagIcon-amSquared-17",
      "ao": "FlagIcon-ao-18",
      "aoSquared": "FlagIcon-aoSquared-19",
      "aq": "FlagIcon-aq-20",
      "aqSquared": "FlagIcon-aqSquared-21",
      "ar": "FlagIcon-ar-22",
      "arSquared": "FlagIcon-arSquared-23",
      "as": "FlagIcon-as-24",
      "asSquared": "FlagIcon-asSquared-25",
      "at": "FlagIcon-at-26",
      "atSquared": "FlagIcon-atSquared-27",
      "au": "FlagIcon-au-28",
      "auSquared": "FlagIcon-auSquared-29",
      "aw": "FlagIcon-aw-30",
      "awSquared": "FlagIcon-awSquared-31",
      "ax": "FlagIcon-ax-32",
      "axSquared": "FlagIcon-axSquared-33",
      "az": "FlagIcon-az-34",
      "azSquared": "FlagIcon-azSquared-35",
      "ba": "FlagIcon-ba-36",
      "baSquared": "FlagIcon-baSquared-37",
      "bb": "FlagIcon-bb-38",
      "bbSquared": "FlagIcon-bbSquared-39",
      "bd": "FlagIcon-bd-40",
      "bdSquared": "FlagIcon-bdSquared-41",
      "be": "FlagIcon-be-42",
      "beSquared": "FlagIcon-beSquared-43",
      "bf": "FlagIcon-bf-44",
      "bfSquared": "FlagIcon-bfSquared-45",
      "bg": "FlagIcon-bg-46",
      "bgSquared": "FlagIcon-bgSquared-47",
      "bh": "FlagIcon-bh-48",
      "bhSquared": "FlagIcon-bhSquared-49",
      "bi": "FlagIcon-bi-50",
      "biSquared": "FlagIcon-biSquared-51",
      "bj": "FlagIcon-bj-52",
      "bjSquared": "FlagIcon-bjSquared-53",
      "bl": "FlagIcon-bl-54",
      "blSquared": "FlagIcon-blSquared-55",
      "bm": "FlagIcon-bm-56",
      "bmSquared": "FlagIcon-bmSquared-57",
      "bn": "FlagIcon-bn-58",
      "bnSquared": "FlagIcon-bnSquared-59",
      "bo": "FlagIcon-bo-60",
      "boSquared": "FlagIcon-boSquared-61",
      "bq": "FlagIcon-bq-62",
      "bqSquared": "FlagIcon-bqSquared-63",
      "br": "FlagIcon-br-64",
      "brSquared": "FlagIcon-brSquared-65",
      "bs": "FlagIcon-bs-66",
      "bsSquared": "FlagIcon-bsSquared-67",
      "bt": "FlagIcon-bt-68",
      "btSquared": "FlagIcon-btSquared-69",
      "bv": "FlagIcon-bv-70",
      "bvSquared": "FlagIcon-bvSquared-71",
      "bw": "FlagIcon-bw-72",
      "bwSquared": "FlagIcon-bwSquared-73",
      "by": "FlagIcon-by-74",
      "bySquared": "FlagIcon-bySquared-75",
      "bz": "FlagIcon-bz-76",
      "bzSquared": "FlagIcon-bzSquared-77",
      "ca": "FlagIcon-ca-78",
      "caSquared": "FlagIcon-caSquared-79",
      "cc": "FlagIcon-cc-80",
      "ccSquared": "FlagIcon-ccSquared-81",
      "cd": "FlagIcon-cd-82",
      "cdSquared": "FlagIcon-cdSquared-83",
      "cf": "FlagIcon-cf-84",
      "cfSquared": "FlagIcon-cfSquared-85",
      "cg": "FlagIcon-cg-86",
      "cgSquared": "FlagIcon-cgSquared-87",
      "ch": "FlagIcon-ch-88",
      "chSquared": "FlagIcon-chSquared-89",
      "ci": "FlagIcon-ci-90",
      "ciSquared": "FlagIcon-ciSquared-91",
      "ck": "FlagIcon-ck-92",
      "ckSquared": "FlagIcon-ckSquared-93",
      "cl": "FlagIcon-cl-94",
      "clSquared": "FlagIcon-clSquared-95",
      "cm": "FlagIcon-cm-96",
      "cmSquared": "FlagIcon-cmSquared-97",
      "cn": "FlagIcon-cn-98",
      "cnSquared": "FlagIcon-cnSquared-99",
      "co": "FlagIcon-co-100",
      "coSquared": "FlagIcon-coSquared-101",
      "cr": "FlagIcon-cr-102",
      "crSquared": "FlagIcon-crSquared-103",
      "cu": "FlagIcon-cu-104",
      "cuSquared": "FlagIcon-cuSquared-105",
      "cv": "FlagIcon-cv-106",
      "cvSquared": "FlagIcon-cvSquared-107",
      "cw": "FlagIcon-cw-108",
      "cwSquared": "FlagIcon-cwSquared-109",
      "cx": "FlagIcon-cx-110",
      "cxSquared": "FlagIcon-cxSquared-111",
      "cy": "FlagIcon-cy-112",
      "cySquared": "FlagIcon-cySquared-113",
      "cz": "FlagIcon-cz-114",
      "czSquared": "FlagIcon-czSquared-115",
      "de": "FlagIcon-de-116",
      "deSquared": "FlagIcon-deSquared-117",
      "dj": "FlagIcon-dj-118",
      "djSquared": "FlagIcon-djSquared-119",
      "dk": "FlagIcon-dk-120",
      "dkSquared": "FlagIcon-dkSquared-121",
      "dm": "FlagIcon-dm-122",
      "dmSquared": "FlagIcon-dmSquared-123",
      "do": "FlagIcon-do-124",
      "doSquared": "FlagIcon-doSquared-125",
      "dz": "FlagIcon-dz-126",
      "dzSquared": "FlagIcon-dzSquared-127",
      "ec": "FlagIcon-ec-128",
      "ecSquared": "FlagIcon-ecSquared-129",
      "ee": "FlagIcon-ee-130",
      "eeSquared": "FlagIcon-eeSquared-131",
      "eg": "FlagIcon-eg-132",
      "egSquared": "FlagIcon-egSquared-133",
      "eh": "FlagIcon-eh-134",
      "ehSquared": "FlagIcon-ehSquared-135",
      "en": "FlagIcon-en-470",
      "enSquared": "FlagIcon-enSquared-471",
      "er": "FlagIcon-er-136",
      "erSquared": "FlagIcon-erSquared-137",
      "es": "FlagIcon-es-138",
      "es-ca": "FlagIcon-es-ca-504",
      "es-caSquared": "FlagIcon-es-caSquared-505",
      "esSquared": "FlagIcon-esSquared-139",
      "et": "FlagIcon-et-140",
      "etSquared": "FlagIcon-etSquared-141",
      "eu": "FlagIcon-eu-506",
      "euSquared": "FlagIcon-euSquared-507",
      "fi": "FlagIcon-fi-142",
      "fiSquared": "FlagIcon-fiSquared-143",
      "fj": "FlagIcon-fj-144",
      "fjSquared": "FlagIcon-fjSquared-145",
      "fk": "FlagIcon-fk-146",
      "fkSquared": "FlagIcon-fkSquared-147",
      "flagIcon": "FlagIcon-flagIcon-2",
      "flagIconBackground": "FlagIcon-flagIconBackground-1",
      "flagIconSquared": "FlagIcon-flagIconSquared-3",
      "fm": "FlagIcon-fm-148",
      "fmSquared": "FlagIcon-fmSquared-149",
      "fo": "FlagIcon-fo-150",
      "foSquared": "FlagIcon-foSquared-151",
      "fr": "FlagIcon-fr-152",
      "frSquared": "FlagIcon-frSquared-153",
      "ga": "FlagIcon-ga-154",
      "gaSquared": "FlagIcon-gaSquared-155",
      "gb": "FlagIcon-gb-156",
      "gb-eng": "FlagIcon-gb-eng-508",
      "gb-engSquared": "FlagIcon-gb-engSquared-509",
      "gb-nir": "FlagIcon-gb-nir-510",
      "gb-nirSquared": "FlagIcon-gb-nirSquared-511",
      "gb-sct": "FlagIcon-gb-sct-512",
      "gb-sctSquared": "FlagIcon-gb-sctSquared-513",
      "gb-wls": "FlagIcon-gb-wls-514",
      "gb-wlsSquared": "FlagIcon-gb-wlsSquared-515",
      "gbSquared": "FlagIcon-gbSquared-157",
      "gd": "FlagIcon-gd-158",
      "gdSquared": "FlagIcon-gdSquared-159",
      "ge": "FlagIcon-ge-160",
      "geSquared": "FlagIcon-geSquared-161",
      "gf": "FlagIcon-gf-162",
      "gfSquared": "FlagIcon-gfSquared-163",
      "gg": "FlagIcon-gg-164",
      "ggSquared": "FlagIcon-ggSquared-165",
      "gh": "FlagIcon-gh-166",
      "ghSquared": "FlagIcon-ghSquared-167",
      "gi": "FlagIcon-gi-168",
      "giSquared": "FlagIcon-giSquared-169",
      "gl": "FlagIcon-gl-170",
      "glSquared": "FlagIcon-glSquared-171",
      "gm": "FlagIcon-gm-172",
      "gmSquared": "FlagIcon-gmSquared-173",
      "gn": "FlagIcon-gn-174",
      "gnSquared": "FlagIcon-gnSquared-175",
      "gp": "FlagIcon-gp-176",
      "gpSquared": "FlagIcon-gpSquared-177",
      "gq": "FlagIcon-gq-178",
      "gqSquared": "FlagIcon-gqSquared-179",
      "gr": "FlagIcon-gr-180",
      "grSquared": "FlagIcon-grSquared-181",
      "gs": "FlagIcon-gs-182",
      "gsSquared": "FlagIcon-gsSquared-183",
      "gt": "FlagIcon-gt-184",
      "gtSquared": "FlagIcon-gtSquared-185",
      "gu": "FlagIcon-gu-186",
      "guSquared": "FlagIcon-guSquared-187",
      "gw": "FlagIcon-gw-188",
      "gwSquared": "FlagIcon-gwSquared-189",
      "gy": "FlagIcon-gy-190",
      "gySquared": "FlagIcon-gySquared-191",
      "hk": "FlagIcon-hk-192",
      "hkSquared": "FlagIcon-hkSquared-193",
      "hm": "FlagIcon-hm-194",
      "hmSquared": "FlagIcon-hmSquared-195",
      "hn": "FlagIcon-hn-196",
      "hnSquared": "FlagIcon-hnSquared-197",
      "hr": "FlagIcon-hr-198",
      "hrSquared": "FlagIcon-hrSquared-199",
      "ht": "FlagIcon-ht-200",
      "htSquared": "FlagIcon-htSquared-201",
      "hu": "FlagIcon-hu-202",
      "huSquared": "FlagIcon-huSquared-203",
      "id": "FlagIcon-id-204",
      "idSquared": "FlagIcon-idSquared-205",
      "ie": "FlagIcon-ie-206",
      "ieSquared": "FlagIcon-ieSquared-207",
      "il": "FlagIcon-il-208",
      "ilSquared": "FlagIcon-ilSquared-209",
      "im": "FlagIcon-im-210",
      "imSquared": "FlagIcon-imSquared-211",
      "in": "FlagIcon-in-212",
      "inSquared": "FlagIcon-inSquared-213",
      "io": "FlagIcon-io-214",
      "ioSquared": "FlagIcon-ioSquared-215",
      "iq": "FlagIcon-iq-216",
      "iqSquared": "FlagIcon-iqSquared-217",
      "ir": "FlagIcon-ir-218",
      "irSquared": "FlagIcon-irSquared-219",
      "is": "FlagIcon-is-220",
      "isSquared": "FlagIcon-isSquared-221",
      "it": "FlagIcon-it-222",
      "itSquared": "FlagIcon-itSquared-223",
      "je": "FlagIcon-je-224",
      "jeSquared": "FlagIcon-jeSquared-225",
      "jm": "FlagIcon-jm-226",
      "jmSquared": "FlagIcon-jmSquared-227",
      "jo": "FlagIcon-jo-228",
      "joSquared": "FlagIcon-joSquared-229",
      "jp": "FlagIcon-jp-230",
      "jpSquared": "FlagIcon-jpSquared-231",
      "ke": "FlagIcon-ke-232",
      "keSquared": "FlagIcon-keSquared-233",
      "kg": "FlagIcon-kg-234",
      "kgSquared": "FlagIcon-kgSquared-235",
      "kh": "FlagIcon-kh-236",
      "khSquared": "FlagIcon-khSquared-237",
      "ki": "FlagIcon-ki-238",
      "kiSquared": "FlagIcon-kiSquared-239",
      "km": "FlagIcon-km-240",
      "kmSquared": "FlagIcon-kmSquared-241",
      "kn": "FlagIcon-kn-242",
      "knSquared": "FlagIcon-knSquared-243",
      "kp": "FlagIcon-kp-244",
      "kpSquared": "FlagIcon-kpSquared-245",
      "kr": "FlagIcon-kr-246",
      "krSquared": "FlagIcon-krSquared-247",
      "kw": "FlagIcon-kw-248",
      "kwSquared": "FlagIcon-kwSquared-249",
      "ky": "FlagIcon-ky-250",
      "kySquared": "FlagIcon-kySquared-251",
      "kz": "FlagIcon-kz-252",
      "kzSquared": "FlagIcon-kzSquared-253",
      "la": "FlagIcon-la-254",
      "laSquared": "FlagIcon-laSquared-255",
      "lb": "FlagIcon-lb-256",
      "lbSquared": "FlagIcon-lbSquared-257",
      "lc": "FlagIcon-lc-258",
      "lcSquared": "FlagIcon-lcSquared-259",
      "li": "FlagIcon-li-260",
      "liSquared": "FlagIcon-liSquared-261",
      "lk": "FlagIcon-lk-262",
      "lkSquared": "FlagIcon-lkSquared-263",
      "lr": "FlagIcon-lr-264",
      "lrSquared": "FlagIcon-lrSquared-265",
      "ls": "FlagIcon-ls-266",
      "lsSquared": "FlagIcon-lsSquared-267",
      "lt": "FlagIcon-lt-268",
      "ltSquared": "FlagIcon-ltSquared-269",
      "lu": "FlagIcon-lu-270",
      "luSquared": "FlagIcon-luSquared-271",
      "lv": "FlagIcon-lv-272",
      "lvSquared": "FlagIcon-lvSquared-273",
      "ly": "FlagIcon-ly-274",
      "lySquared": "FlagIcon-lySquared-275",
      "ma": "FlagIcon-ma-276",
      "maSquared": "FlagIcon-maSquared-277",
      "mc": "FlagIcon-mc-278",
      "mcSquared": "FlagIcon-mcSquared-279",
      "md": "FlagIcon-md-280",
      "mdSquared": "FlagIcon-mdSquared-281",
      "me": "FlagIcon-me-282",
      "meSquared": "FlagIcon-meSquared-283",
      "mf": "FlagIcon-mf-284",
      "mfSquared": "FlagIcon-mfSquared-285",
      "mg": "FlagIcon-mg-286",
      "mgSquared": "FlagIcon-mgSquared-287",
      "mh": "FlagIcon-mh-288",
      "mhSquared": "FlagIcon-mhSquared-289",
      "mk": "FlagIcon-mk-290",
      "mkSquared": "FlagIcon-mkSquared-291",
      "ml": "FlagIcon-ml-292",
      "mlSquared": "FlagIcon-mlSquared-293",
      "mm": "FlagIcon-mm-294",
      "mmSquared": "FlagIcon-mmSquared-295",
      "mn": "FlagIcon-mn-296",
      "mnSquared": "FlagIcon-mnSquared-297",
      "mo": "FlagIcon-mo-298",
      "moSquared": "FlagIcon-moSquared-299",
      "mp": "FlagIcon-mp-300",
      "mpSquared": "FlagIcon-mpSquared-301",
      "mq": "FlagIcon-mq-302",
      "mqSquared": "FlagIcon-mqSquared-303",
      "mr": "FlagIcon-mr-304",
      "mrSquared": "FlagIcon-mrSquared-305",
      "ms": "FlagIcon-ms-306",
      "msSquared": "FlagIcon-msSquared-307",
      "mt": "FlagIcon-mt-308",
      "mtSquared": "FlagIcon-mtSquared-309",
      "mu": "FlagIcon-mu-310",
      "muSquared": "FlagIcon-muSquared-311",
      "mv": "FlagIcon-mv-312",
      "mvSquared": "FlagIcon-mvSquared-313",
      "mw": "FlagIcon-mw-314",
      "mwSquared": "FlagIcon-mwSquared-315",
      "mx": "FlagIcon-mx-316",
      "mxSquared": "FlagIcon-mxSquared-317",
      "my": "FlagIcon-my-318",
      "mySquared": "FlagIcon-mySquared-319",
      "mz": "FlagIcon-mz-320",
      "mzSquared": "FlagIcon-mzSquared-321",
      "na": "FlagIcon-na-322",
      "naSquared": "FlagIcon-naSquared-323",
      "nc": "FlagIcon-nc-324",
      "ncSquared": "FlagIcon-ncSquared-325",
      "ne": "FlagIcon-ne-326",
      "neSquared": "FlagIcon-neSquared-327",
      "nf": "FlagIcon-nf-328",
      "nfSquared": "FlagIcon-nfSquared-329",
      "ng": "FlagIcon-ng-330",
      "ngSquared": "FlagIcon-ngSquared-331",
      "ni": "FlagIcon-ni-332",
      "niSquared": "FlagIcon-niSquared-333",
      "nl": "FlagIcon-nl-334",
      "nlSquared": "FlagIcon-nlSquared-335",
      "no": "FlagIcon-no-336",
      "noSquared": "FlagIcon-noSquared-337",
      "np": "FlagIcon-np-338",
      "npSquared": "FlagIcon-npSquared-339",
      "nr": "FlagIcon-nr-340",
      "nrSquared": "FlagIcon-nrSquared-341",
      "nu": "FlagIcon-nu-342",
      "nuSquared": "FlagIcon-nuSquared-343",
      "nz": "FlagIcon-nz-344",
      "nzSquared": "FlagIcon-nzSquared-345",
      "om": "FlagIcon-om-346",
      "omSquared": "FlagIcon-omSquared-347",
      "pa": "FlagIcon-pa-348",
      "paSquared": "FlagIcon-paSquared-349",
      "pe": "FlagIcon-pe-350",
      "peSquared": "FlagIcon-peSquared-351",
      "pf": "FlagIcon-pf-352",
      "pfSquared": "FlagIcon-pfSquared-353",
      "pg": "FlagIcon-pg-354",
      "pgSquared": "FlagIcon-pgSquared-355",
      "ph": "FlagIcon-ph-356",
      "phSquared": "FlagIcon-phSquared-357",
      "pk": "FlagIcon-pk-358",
      "pkSquared": "FlagIcon-pkSquared-359",
      "pl": "FlagIcon-pl-360",
      "plSquared": "FlagIcon-plSquared-361",
      "pm": "FlagIcon-pm-362",
      "pmSquared": "FlagIcon-pmSquared-363",
      "pn": "FlagIcon-pn-364",
      "pnSquared": "FlagIcon-pnSquared-365",
      "pr": "FlagIcon-pr-366",
      "prSquared": "FlagIcon-prSquared-367",
      "ps": "FlagIcon-ps-368",
      "psSquared": "FlagIcon-psSquared-369",
      "pt": "FlagIcon-pt-370",
      "ptSquared": "FlagIcon-ptSquared-371",
      "pw": "FlagIcon-pw-372",
      "pwSquared": "FlagIcon-pwSquared-373",
      "py": "FlagIcon-py-374",
      "pySquared": "FlagIcon-pySquared-375",
      "qa": "FlagIcon-qa-376",
      "qaSquared": "FlagIcon-qaSquared-377",
      "re": "FlagIcon-re-378",
      "reSquared": "FlagIcon-reSquared-379",
      "ro": "FlagIcon-ro-380",
      "roSquared": "FlagIcon-roSquared-381",
      "rs": "FlagIcon-rs-382",
      "rsSquared": "FlagIcon-rsSquared-383",
      "ru": "FlagIcon-ru-384",
      "ruSquared": "FlagIcon-ruSquared-385",
      "rw": "FlagIcon-rw-386",
      "rwSquared": "FlagIcon-rwSquared-387",
      "sa": "FlagIcon-sa-388",
      "saSquared": "FlagIcon-saSquared-389",
      "sb": "FlagIcon-sb-390",
      "sbSquared": "FlagIcon-sbSquared-391",
      "sc": "FlagIcon-sc-392",
      "scSquared": "FlagIcon-scSquared-393",
      "sd": "FlagIcon-sd-394",
      "sdSquared": "FlagIcon-sdSquared-395",
      "se": "FlagIcon-se-396",
      "seSquared": "FlagIcon-seSquared-397",
      "sg": "FlagIcon-sg-398",
      "sgSquared": "FlagIcon-sgSquared-399",
      "sh": "FlagIcon-sh-400",
      "shSquared": "FlagIcon-shSquared-401",
      "si": "FlagIcon-si-402",
      "siSquared": "FlagIcon-siSquared-403",
      "sj": "FlagIcon-sj-404",
      "sjSquared": "FlagIcon-sjSquared-405",
      "sk": "FlagIcon-sk-406",
      "skSquared": "FlagIcon-skSquared-407",
      "sl": "FlagIcon-sl-408",
      "slSquared": "FlagIcon-slSquared-409",
      "sm": "FlagIcon-sm-410",
      "smSquared": "FlagIcon-smSquared-411",
      "sn": "FlagIcon-sn-412",
      "snSquared": "FlagIcon-snSquared-413",
      "so": "FlagIcon-so-414",
      "soSquared": "FlagIcon-soSquared-415",
      "sr": "FlagIcon-sr-416",
      "srSquared": "FlagIcon-srSquared-417",
      "ss": "FlagIcon-ss-418",
      "ssSquared": "FlagIcon-ssSquared-419",
      "st": "FlagIcon-st-420",
      "stSquared": "FlagIcon-stSquared-421",
      "sv": "FlagIcon-sv-422",
      "svSquared": "FlagIcon-svSquared-423",
      "sx": "FlagIcon-sx-424",
      "sxSquared": "FlagIcon-sxSquared-425",
      "sy": "FlagIcon-sy-426",
      "sySquared": "FlagIcon-sySquared-427",
      "sz": "FlagIcon-sz-428",
      "szSquared": "FlagIcon-szSquared-429",
      "tc": "FlagIcon-tc-430",
      "tcSquared": "FlagIcon-tcSquared-431",
      "td": "FlagIcon-td-432",
      "tdSquared": "FlagIcon-tdSquared-433",
      "tf": "FlagIcon-tf-434",
      "tfSquared": "FlagIcon-tfSquared-435",
      "tg": "FlagIcon-tg-436",
      "tgSquared": "FlagIcon-tgSquared-437",
      "th": "FlagIcon-th-438",
      "thSquared": "FlagIcon-thSquared-439",
      "tj": "FlagIcon-tj-440",
      "tjSquared": "FlagIcon-tjSquared-441",
      "tk": "FlagIcon-tk-442",
      "tkSquared": "FlagIcon-tkSquared-443",
      "tl": "FlagIcon-tl-444",
      "tlSquared": "FlagIcon-tlSquared-445",
      "tm": "FlagIcon-tm-446",
      "tmSquared": "FlagIcon-tmSquared-447",
      "tn": "FlagIcon-tn-448",
      "tnSquared": "FlagIcon-tnSquared-449",
      "to": "FlagIcon-to-450",
      "toSquared": "FlagIcon-toSquared-451",
      "tr": "FlagIcon-tr-452",
      "trSquared": "FlagIcon-trSquared-453",
      "tt": "FlagIcon-tt-454",
      "ttSquared": "FlagIcon-ttSquared-455",
      "tv": "FlagIcon-tv-456",
      "tvSquared": "FlagIcon-tvSquared-457",
      "tw": "FlagIcon-tw-458",
      "twSquared": "FlagIcon-twSquared-459",
      "tz": "FlagIcon-tz-460",
      "tzSquared": "FlagIcon-tzSquared-461",
      "ua": "FlagIcon-ua-462",
      "uaSquared": "FlagIcon-uaSquared-463",
      "ug": "FlagIcon-ug-464",
      "ugSquared": "FlagIcon-ugSquared-465",
      "um": "FlagIcon-um-466",
      "umSquared": "FlagIcon-umSquared-467",
      "un": "FlagIcon-un-516",
      "unSquared": "FlagIcon-unSquared-517",
      "us": "FlagIcon-us-468",
      "usSquared": "FlagIcon-usSquared-469",
      "uy": "FlagIcon-uy-472",
      "uySquared": "FlagIcon-uySquared-473",
      "uz": "FlagIcon-uz-474",
      "uzSquared": "FlagIcon-uzSquared-475",
      "va": "FlagIcon-va-476",
      "vaSquared": "FlagIcon-vaSquared-477",
      "vc": "FlagIcon-vc-478",
      "vcSquared": "FlagIcon-vcSquared-479",
      "ve": "FlagIcon-ve-480",
      "veSquared": "FlagIcon-veSquared-481",
      "vg": "FlagIcon-vg-482",
      "vgSquared": "FlagIcon-vgSquared-483",
      "vi": "FlagIcon-vi-484",
      "viSquared": "FlagIcon-viSquared-485",
      "vn": "FlagIcon-vn-486",
      "vnSquared": "FlagIcon-vnSquared-487",
      "vu": "FlagIcon-vu-488",
      "vuSquared": "FlagIcon-vuSquared-489",
      "wf": "FlagIcon-wf-490",
      "wfSquared": "FlagIcon-wfSquared-491",
      "ws": "FlagIcon-ws-492",
      "wsSquared": "FlagIcon-wsSquared-493",
      "xk": "FlagIcon-xk-518",
      "xkSquared": "FlagIcon-xkSquared-519",
      "ye": "FlagIcon-ye-494",
      "yeSquared": "FlagIcon-yeSquared-495",
      "yt": "FlagIcon-yt-496",
      "ytSquared": "FlagIcon-ytSquared-497",
      "za": "FlagIcon-za-498",
      "zaSquared": "FlagIcon-zaSquared-499",
      "zm": "FlagIcon-zm-500",
      "zmSquared": "FlagIcon-zmSquared-501",
      "zw": "FlagIcon-zw-502",
      "zwSquared": "FlagIcon-zwSquared-503",
    }
  }
  code="en"
  square={false}
/>
`;

exports[`<FlagIconCss /> Should render properly 1`] = `
<span
  className="MuiBox-root MuiBox-root-521 FlagIcon-flagIcon-2 FlagIcon-en-470"
/>
`;
